src/js/lib/photo-swipe.js

```diff
@@ -20,12 +19,13 @@ const members = new Vue({
       { name: 'joseph', image: '/images/members/8.jpg' },
       { name: 'celso', image: '/images/members/9.jpg' },
       { name: 'becker', image: '/images/members/10.jpg' },
-      { name: 'mario', image: '/images/members/11.jpg' },
       { name: 'sebas', image: '/images/members/12.jpg' },
       { name: 'ban', image: '/images/members/13.jpg' },
       { name: 'rafa', image: '/images/members/14.jpg' },
-      { name: 'denis', image: '/images/members/15.jpg' },
       { name: 'helal', image: '/images/members/16.jpg' },
+      { name: 'marcos', image: '/images/members/17.jpg' },
+      { name: 'recidive', image: '/images/members/1.jpg' },
```

Comment:
And is it really needed? As far as I understand it, the current mosaic is quite flexible. We may need to adjust a bit of css, but that's all.